frontend/src/metabase/ui/components/theme/ThemeProvider/ThemeProvider.tsx

```diff
@@ -76,6 +76,8 @@ export const ThemeProvider = (props: ThemeProviderProps) => {
       getStyleNonce={() => window.MetabaseNonce ?? "metabase"}
       classNamesPrefix="mb-mantine"
       cssVariablesSelector={isEmbeddingSdk ? ".mb-wrapper" : undefined}
+      // If we try to inject CSS variables to `.mb-wrapper`, it will slow the Jest tests down like crazy.
+      withCssVariables={!isJest}
```

Comment:
instead of putting test-specific code in the theme provider itself, can you expose this as a prop and then set it in the `renderWithPRoviders` / `renderWithSdkProviders` wrappers

Reply:
Yeah, good point. I was just trying to get it working the whole week last week, so it wasn't the prettiest 😅

Reply:
Addressed in https://github.com/metabase/metabase/pull/60049/commits/b3a4eab74155ed3b3d40072d56b8479788232307.

I found we already accept `mantineProviderProps` that did that in non-SDK render functions.
https://github.com/metabase/metabase/blob/b3a4eab74155ed3b3d40072d56b8479788232307/frontend/test/__support__/ui.tsx#L250

So, I do the same for the SDK here.